src/test/java/com/andersen/TicketService/TicketServiceTest.java

```diff
@@ -0,0 +1,197 @@
+package com.andersen.TicketService;
+
+import org.example.DAOs.TicketDAO;
+import org.example.entities.Client;
+import org.example.entities.Ticket;
+import org.example.enums.StadiumSectors;
+import org.example.services.TicketService;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.math.BigDecimal;
+import java.util.Collections;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+class TicketServiceTest {
+
+    private TicketDAO ticketDAO;
+    private TicketService ticketService;
+
+    // Setup for each test. Initializes mock objects and the service under test.
+    @BeforeEach
+    void setup() {
+        ticketDAO = mock(TicketDAO.class);
+        ticketService = new TicketService(ticketDAO);
+    }
+
+    // Test for adding a ticket. Includes positive, negative, and corner cases.
+    @Test
+    void testAddTicket() {
```

Comment:
for each case create please separate test, also don't repeate 'test' in method name inside Test class